src/main/java/br/com/zup/concessionariacitroen/services/EstoqueService.java

```diff
@@ -0,0 +1,20 @@
+package br.com.zup.concessionariacitroen.services;
+
+import br.com.zup.concessionariacitroen.models.Automovel;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EstoqueService {
+
+    private static List<Automovel> ListaAutomoveis = new ArrayList<>();
```

Comment:
O alterar o tipo da lista, e o nome da variável.
```suggestion
    private static List<Carro> carros = new ArrayList<>();
```